### ISSUE TITLE
IPHONEOS_DEPLOYMENT_TARGET Warnings in Xcode 12

### ISSUE THREAD

**Issue by jchitel** [Quick/Nimble#808]
- [x] I have read [CONTRIBUTING](https://github.com/Quick/Nimble/blob/master/CONTRIBUTING.md) and have done my best to follow them.

### What did you do?

Add Nimble as a dependency via Swift Package Manager, and build my tests.

### What did you expect to happen?

No warnings related to external code.

### What actually happened instead?

I receive several warnings, all of which have the content:

```
The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99.
```

**This variable is indeed set to 8.0 in `Nimble.xcodeproj/project.pbxproj`, so that is certainly the culprit. It appears that Xcode only supports 5 versions prior to the current version. As the latest iOS version for Xcode 12 is 14, this means the earliest version that they will support is 9.0. Considering that both [Apple](https://developer.apple.com/support/app-store/) and [StatCounter](https://gs.statcounter.com/os-version-market-share/ios/mobile-tablet/worldwide) no longer even display statistics for any version prior to iOS 10, I think it is probably safe for Nimble to bump this up to at least iOS 9.**

### Environment

List the software versions you're using:

- Quick: not using Quick
- Nimble: *8.1.1*
- Xcode Version: *12.0 beta 4 (12A8179i)*
- Swift Version: *Xcode Default* (Should be 5.x?)

Please also mention which package manager you used and its version. Delete the
other package managers in this list:

- Swift Package Manager *Swift Package Manager - Swift 5.3.0*

### Project that demonstrates the issue

This will likely occur with any project using Nimble in Xcode 12.

**Comment by arangato** [comment]
PR https://github.com/Quick/Nimble/pull/813 should solve it.

**Comment by ikesyo** [comment]
Closed by #825.
